src/main/java/Duke.java

```diff
@@ -1,10 +1,80 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static final String COMMAND_LIST = "list";
```

Comment:
I think upper case is used for constants and strings are immutable by default. Thus, maybe can change the naming of the variables to fit coding standard.